models/caseloads.js

```diff
@@ -0,0 +1,69 @@
+const BaseModel = require("./base/dynamo-db");
+const uuid = require("uuid");
+class CohortModel extends BaseModel {
```

Comment:
there is already a CohortModel in cohort.js. Is this needed/used? Does it need renamed? this seems to point to the caseloads table which is already covered in caseload.js

Reply:
Sorry I renamed the model and it must've left the old one behind.